test/test.js

```diff
@@ -10,8 +13,31 @@ jasmine.loadConfig({
     ]
 });
 
-var runner =  new JsReporters.JasmineAdapter(jasmine);
-var testReporter = new JsReporters.TestReporter(runner, referenceData.Jasmine);
+var jasmineRunner = new JsReporters.JasmineAdapter(jasmine.env);
+jasmine.addReporter({}); // Suppress the default reporter
+var jasmineTestReporter = new JsReporters.TestReporter(jasmineRunner, referenceData.Jasmine);
 
 jasmine.execute();
-process.exit(testReporter.ok ? 0 : 1);
+if (!jasmineTestReporter.ok) {
+    throw new Error("Tests for JasmineAdapter failed!");
+}
+
+//TODO: Use a proper test framework?
```

Comment:
I'm not sure what are we talking about, but I have some opinions from what I can see and presume:

I can't see how throwing an error on this module is better than a direct call on the process.exit if the idea is to exit. The error above on line 22 (`"Tests for JasmineAdapter failed!"`) seems weird to me.

Another issue I see on this file is that we are loading 2 reporters, one for Jasmine and one for QUnit, I wonder if they work better on individual files.

Following the file split, I'm also thinking about a proper reaction if `jasmineTestReporter.ok` or `qunitTestReporter.ok` are `false`. They can be improved to something that doesn't interrupt the tests, following to whatever comes next.